editor/editor_node.cpp

```diff
@@ -4338,6 +4343,91 @@ void EditorNode::progress_end_task_bg(const String &p_task) {
 	singleton->progress_hb->end_task(p_task);
 }
 
+String EditorNode::_get_system_info() const {
+	String distribution_name = OS::get_singleton()->get_distribution_name();
+	if (distribution_name.is_empty()) {
+		distribution_name = OS::get_singleton()->get_name();
+	}
+	if (distribution_name.is_empty()) {
+		distribution_name = "Other";
+	}
+	const String distribution_version = OS::get_singleton()->get_version();
+
+	const String godot_version = String(VERSION_FULL_BUILD);
+
+	String driver_name_path = "rendering/rendering_device/driver";
+	const String simplified_os_name = OS::get_singleton()->get_name_simplified();
+	if (!simplified_os_name.is_empty()) {
+		const Vector<String> valid_overrides = { "linuxbsd", "windows", "android", "ios", "macos" };
+		if (valid_overrides.find(simplified_os_name) != -1) {
+			driver_name_path += "." + simplified_os_name; // Respect platform-specific driver override.
+		}
+	}
```

Comment:
You don't need to do that, feature overrides are resolved automatically.
You should just do `ProjectSettings::get_singleton()->get("rendering/rendering_device/driver")` and you'll get the value that matches the platform. (Or `GLOBAL_GET("rendering/rendering_device/driver")` which is the more common alias.)